pkgs/by-name/bu/buttermanager/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, python3
+, qt5
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "buttermanager";
+  version = "2.5.0";
```

Comment:
https://github.com/egara/buttermanager/releases/tag/2.5.1 got already released ✌🏻

Reply:
To be expected, taking 3 months to merge already.
Thanks for the heads up! Updated.